charger/eebus.go

```diff
@@ -515,18 +536,16 @@ func (c *EEBus) CurrentPower() (float64, error) {
 	var power float64
 	var phase uint
 	for phase = 1; phase <= data.EVData.ConnectedPhases; phase++ {
```

Comment:
Slice mit Start bei 1? In Go wäre 0 üblich.

    for phase := uint(0)

Reply:
Hier wird eine Map genutzt, kein Slice